packages/fx-core/src/plugins/resource/bot/v2/plugin.ts

```diff
@@ -180,19 +191,15 @@ export class TeamsBotV2Impl {
   }
 
   private getBicepConfigs(ctx: Context, inputs: Inputs): BicepConfigs {
-    const lang = this.resolveProgrammingLanguage(ctx);
-
+    const languageMapping: { [key: string]: string } = {
```

Comment:
This map could be a const outside this method, so that it won't be created again and again when this method is called. It could be somewhere bot plugin defines const.

The map name could be runtimeMap